src/editor/components/flow/edit-dialog.tsx

```diff
@@ -0,0 +1,68 @@
+import { useState } from 'react'
+import { Dialog } from '../dialog'
+import { Button } from '../form/button'
+import { Input } from '../form/input'
+import { Select } from '../form/select'
+
+interface EditDataFlowDialogProps {
+  isEditOpen: boolean,
+  setIsEditOpen( value: boolean ): void
+  save(): void
+  flow: any
+  setFlow( value: any ): void
+  deleteFlow(): void
+}
+
+const states = [
+  { value: 'Active', name: 'Active' },
+  { value: 'Disabled', name: 'Disabled' }
+]
```

Comment:
maybe this should be defined somewhere else so other components using the types uf states can access it

Reply:
Yes, this should be an ENUM at some point.